components/script/dom/htmlimageelement.rs

```diff
@@ -3,6 +3,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use app_units::{Au, AU_PER_PX};
+use core::ops::Deref;
```

Comment:
An explicit call to deref should not be necessary, if `let unparsed_sizes = input.split(',').collect::<Vec<_>>();` does not work, try `let unparsed_sizes = (*input).split(',').collect::<Vec<_>>();`.